From 7b19581111079aff57954e0a0bf189f17400712a Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Sun, 22 Jan 2017 20:45:08 +1300
Subject: [PATCH 19/27] Fix EUMM podlocal tests

---
 cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t b/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
index 31d645be5e..95564cc3c6 100644
--- a/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
+++ b/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
@@ -22,7 +22,7 @@ use Test::More
     !MM->can_run(make()) && $ENV{PERL_CORE} && $Config{'usecrosscompile'}
     ? (skip_all => "cross-compiling and make not available")
     : ();
-plan tests => 4 + $CLEANUP + @INSTDIRS * (15 + $CLEANUP);
+plan tests => 4 + $CLEANUP + @INSTDIRS * (13 + $CLEANUP);
 
 my $Is_VMS = $^O eq 'VMS';
 
@@ -72,7 +72,6 @@ for my $instdir (@INSTDIRS) {
     ("$instdir/lib/perl5/Big/Dummy.pm",
      "$instdir/lib/perl5/Big/Liar.pm",
      "$instdir/bin/program",
-     "$instdir/lib/perl5/$Config{archname}/perllocal.pod",
      "$instdir/lib/perl5/$Config{archname}/auto/Big/Dummy/.packlist"
     );
 
-- 
2.39.3

