From 0e1dc8276d4ac500a40381ccc94bbe81c3d746b7 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Sun, 22 Jan 2017 20:45:08 +1300
Subject: [PATCH 19/30] Fix EUMM podlocal tests

---
 cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t b/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
index 69398117b7..5b84a0e9e1 100644
--- a/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
+++ b/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
@@ -21,7 +21,7 @@ use Test::More
     !MM->can_run(make()) && $ENV{PERL_CORE} && $Config{'usecrosscompile'}
     ? (skip_all => "cross-compiling and make not available")
     : ();
-plan tests => 4 + $CLEANUP + @INSTDIRS * (15 + $CLEANUP);
+plan tests => 4 + $CLEANUP + @INSTDIRS * (13 + $CLEANUP);
 
 my $Is_VMS = $^O eq 'VMS';
 
@@ -71,7 +71,6 @@ for my $instdir (@INSTDIRS) {
     ("$instdir/lib/perl5/Big/Dummy.pm",
      "$instdir/lib/perl5/Big/Liar.pm",
      "$instdir/bin/program",
-     "$instdir/lib/perl5/$Config{archname}/perllocal.pod",
      "$instdir/lib/perl5/$Config{archname}/auto/Big/Dummy/.packlist"
     );
 
-- 
2.27.0

