diff -Naur a/cpan/IO-Socket-IP/t/15io-socket.t b/cpan/IO-Socket-IP/t/15io-socket.t
--- a/cpan/IO-Socket-IP/t/15io-socket.t
+++ b/cpan/IO-Socket-IP/t/15io-socket.t
@@ -4,6 +4,11 @@
 use warnings;
 
 use Test::More;
+BEGIN {
+  if ( $ENV{NO_GENTOO_NETWORK_TESTS} ) {
+  	plan skip_all => "Tests disabled by NO_GENTOO_NETWORK_TESTS";
+  }
+}
 
 use IO::Socket;
 use IO::Socket::IP -register;
diff -Naur a/cpan/IO-Socket-IP/t/16v6only.t b/cpan/IO-Socket-IP/t/16v6only.t
--- perl-5.20.2/cpan/IO-Socket-IP/t/16v6only.t
+++ perl-5.20.2b/cpan/IO-Socket-IP/t/16v6only.t
@@ -4,6 +4,11 @@
 use warnings;
 
 use Test::More;
+BEGIN {
+  if ( $ENV{NO_GENTOO_NETWORK_TESTS} ) {
+        plan skip_all => "Tests disabled by NO_GENTOO_NETWORK_TESTS";
+  }
+}
 
 use Socket qw(PF_INET6 PF_INET IPPROTO_IPV6 IPV6_V6ONLY);
 use IO::Socket::IP;
