Subject: Minimal changes to test framework so everything passes with Gentoo patchset

diff -ruN perl-5.24.0.orig/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t perl-5.24.0/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t
--- perl-5.24.0.orig/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t	2016-03-01 13:33:01.000000000 +0100
+++ perl-5.24.0/cpan/ExtUtils-MakeMaker/t/INSTALL_BASE.t	2016-06-04 23:16:44.306347943 +0200
@@ -21,7 +21,7 @@
     !MM->can_run(make()) && $ENV{PERL_CORE} && $Config{'usecrosscompile'}
     ? (skip_all => "cross-compiling and make not available")
     : ();
-plan tests => 4 + $CLEANUP + @INSTDIRS * (15 + $CLEANUP);
+plan tests => 4 + $CLEANUP + @INSTDIRS * (13 + $CLEANUP);
 
 my $Is_VMS = $^O eq 'VMS';
 
@@ -64,7 +64,6 @@
     ("$instdir/lib/perl5/Big/Dummy.pm",
      "$instdir/lib/perl5/Big/Liar.pm",
      "$instdir/bin/program",
-     "$instdir/lib/perl5/$Config{archname}/perllocal.pod",
      "$instdir/lib/perl5/$Config{archname}/auto/Big/Dummy/.packlist"
     );
 
